src/Schema.php

```diff
@@ -790,11 +794,7 @@ protected function findColumns(TableSchemaInterface $table): bool
                     $loadColumnSchema->defaultValue(new Expression($defaultValue));
                 } elseif ($loadColumnSchema->getType() === 'boolean') {
                     $loadColumnSchema->defaultValue($defaultValue  === 'true');
-                } elseif (is_string($defaultValue) && preg_match("/^B'(.*?)'::/", $defaultValue, $matches)) {
-                    $loadColumnSchema->defaultValue(bindec($matches[1]));
-                } elseif (is_string($defaultValue) && preg_match("/^'(\d+)'::\"bit\"$/", $defaultValue, $matches)) {
```

Comment:
Are you sure? Maybe oldest version return this format of default values? for example 9.0.1 and reading schema of view?

Reply:
Simple show me proof. Without adding tests. I want be sure.

Reply:
> Are you sure? Maybe oldest version return this format of default values? for example 9.0.1 and reading schema of view?

* 9.6.0

Reply:
`} elseif (is_string($defaultValue) && preg_match("/^B?'(.*?)'::/", $defaultValue, $matches)) {`
This check match both `"/^B'(.*?)'::/"` and `"/^'(\d+)'::\"bit\"$/"`

And as the column of type `bit` it will be converted in `ColumnSchema::phpTypecast()`